coordinator/src/routes/dispense/task.ts

```diff
@@ -24,9 +24,9 @@ export default [
         return responseToolkit.response(fhir.createOperationOutcome(issues)).code(400).type(CONTENT_TYPE_FHIR)
```

Comment:
logging updated to return/withdraw but payload still named withdraw

Reply:
Good spot, will update